packages/contracts/src/helpers/sway.ts

```diff
@@ -0,0 +1,22 @@
+import { join } from 'path';
+
+import type { ContractsConfig } from '../types';
+
+export function getFolderName(contractPath: string) {
+  const folderName = contractPath.split('/').slice(-1)[0];
+  return folderName;
+}
+
+export function getBinaryPath(contractPath: string) {
+  const folderName = getFolderName(contractPath);
+  return join(contractPath, `/out/debug/${folderName}.bin`);
```

Comment:
Just something to note as a possible future bug, but I was running into an issue where the `folderName` was not equal to the name of the contract binary and I had to rename the name of the contract in the corresponding `Forc.toml` file in order to get it to work.